### PR TITLE
feat: Help page

### PR DESCRIPTION
**Pull request by yash-rajpal**
## Screenshots
image

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Improvement (non-breaking change which improves a current function)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update (if none of the other choices apply)

## Checklist

- [ ] I have read the [CONTRIBUTING](https://github.com/RocketChat/Rocket.Chat/blob/develop/.github/CONTRIBUTING.md#contributing-to-rocketchat) doc
- [ ] I have signed the [CLA](https://cla-assistant.io/RocketChat/Rocket.Chat.ReactNative)
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if applicable)
- [ ] I have added necessary documentation (if applicable)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

[MA-71](https://rocketchat.atlassian.net/browse/MA-71)

[MA-71]: https://rocketchat.atlassian.net/browse/MA-71?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: app/views/SettingsView/index.tsx [hunk 1]
```diff
@@ -225,7 +225,7 @@ const SettingsView = (): React.ReactElement => {
 					/>
 					<List.Separator />
 					<List.Item
-						title='Media_auto_download'
+					title='Media_auto_download'
```
Comment (diegolmello) [inline app/views/SettingsView/index.tsx:228]: Unwanted indentation

File: app/views/GetHelpView.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+	return (
+		<SafeAreaView>
+			<StatusBar />
+			<List.Container>
+				<List.Section>
+					<List.Separator/>
+					<List.Item
+						title='Rocket_Chat_Documentation'
+						right={() => <List.Icon name='new-window' style={I18nManager.isRTL ? { transform: [{ rotate: '180deg' }] } : null}/>}
```
Comment (diegolmello) [inline app/views/GetHelpView.tsx:35]: Do we need to rotate the icon on RTL?

File: app/i18n/locales/en.json [hunk 3]
```diff
@@ -6,6 +6,7 @@
   "A_new_owner_will_be_assigned_automatically_to__count__room": "A new owner will be assigned automatically to {{count}} room.",
   "A_new_owner_will_be_assigned_automatically_to__count__rooms": "A new owner will be assigned automatically to {{count}} rooms.",
   "accept": "Accept",
+  "Accessibility_statement": "Accessibility statement",
```
Comment (diegolmello) [inline app/i18n/locales/en.json:9]: Can you add i18n for other languages using AI?
It's not perfect, but it's better than not having it translated.

**Review by diegolmello — CHANGES_REQUESTED** [review]
Missing E2E tests

File: e2e/tests/assorted/04-setting.spec.ts [hunk 4]
```diff
@@ -61,6 +61,10 @@ describe('Settings screen', () => {
 			await expect(element(by.id('settings-view-security-privacy'))).toExist();
 		});
 
+		it('should have get help', async () => {
```
[the remark this answers is not in the document]
Comment (yash-rajpal) [reply e2e/tests/assorted/04-setting.spec.ts:64]: kkkk

File: app/views/SettingsView/index.tsx [hunk 5]
```diff
@@ -238,9 +238,17 @@ const SettingsView = (): React.ReactElement => {
...
 						left={() => <List.Icon name='locker' />}
 					/>
+					<List.Separator />
 				</List.Section>
 
 				<List.Section>
+					<List.Item
+						title='Get_help'
+						left={() => <List.Icon name='support' />}
+						right={() => <List.Icon name='new-window' />}
```
[the remark this answers is not in the document]
Comment (yash-rajpal) [reply app/views/SettingsView/index.tsx:248]: I'm not sure, maybe we should as the icon is pointing outwards.

I'll confirm with the design team.

[the remark this answers is not in the document]
Comment (yash-rajpal) [reply app/views/SettingsView/index.tsx]: Confirmed with design team, it should always point outwards, should point left when UI is RTL.

Created a new component and using it everywhere

**Review by diegolmello — APPROVED** [review]

**Pull request merged**